converged_sound_open_fw_modules_development/FW/intel_common/module_binmaps/aca_module.binmap

```diff
@@ -0,0 +1,35 @@
+module o ACA
```

Comment:
@bkokoszx this is very interesting. Can you add more information inthe commit message. 

What is a module? A separate binary with some functionality?

How is this used by sof:
 - statically compiled 
 - dynamically loaded at runtime
etc?

Daniel.

Reply:
> What is a module? A separate binary with some functionality?
> 
> How is this used by sof:
> 
> * statically compiled
> * dynamically loaded at runtime
>   etc?
The module here is an equivalent of SOF processing component. The difference is that these modules will not be statically compiled with SOF base FW but dynamically loaded as a separate binary. That will allow to have one base SOF FW for platform generation and depending on specific audio topology configuration choose which 3rd party processing modules to load.

The modules will be usable with SOF once we implement dynamic library load feature and align processing modules API. This will be the next step.

Reply:
@dbaluta 
The Intel Confidential headers were my fault. I've just updated it.

Reply:
@mwasko This is a great feature. We also have this project for Google Summer of Code. 

https://wiki.linuxfoundation.org/gsoc/2021-gsoc-sound-open-firmware